### ISSUE TITLE
Geolocation

### ISSUE THREAD

**Issue by mlagally** [w3c/wot-profile#137]
The profile should define standard metadata properties for location.
This is required for interoperability with sensors.

**Comment by benfrancis** [comment]
See also: https://w3c.github.io/wot-thing-description/#semantic-annotations-example-geoloc

**Comment by mlagally** [comment]
The current example in the TD does not contain an altitude, should be added there.
The profile could RECOMMEND to use the vocabulary from the TD example:  (https://www.w3.org/2003/01/geo/)

**Comment by benfrancis** [comment]
I agree that recommending a standard way to express geolocation in the Core Profile would aid interoperability.

I think the consensus around the Thing Description specification is that existing external ontologies like schema.org should be used for these kinds of metadata, rather than defining something new in the Thing Description namespace. I agree with this.

Where I think this gets a bit tricky is if we're going to recommend a particular external ontology for semantic annotations for geolocation, should we be recommending a particular ontology for other concepts too? We've already discussed units as another example (#29). Where do we stop?

Schema.org defines ontologies for lots of other concepts that could be relevant to IoT devices, and iotschema.org (whilst not as stable) goes even further. There are also lots of other schema repositories like oneiota.org, onedm.org and webthings.io/schemas.

What consistent criteria could we define to decide what should and shouldn't be recommended as part of the Core Profile?

**Comment by mlagally** [comment]
Deferring to Profile 1.1, since we currently have different opinions on where/how geolocation should be handled.

**Comment by benfrancis** [comment]
Please note closed pull request https://github.com/w3c/wot-profile/pull/264 for when we re-visit this in 1.1.
